docs/run-local.md

```diff
@@ -42,7 +42,7 @@ You may wish to run Rancher rather than the dashboard. Rancher provide a step by
 
 ## Add Dependencies
 
-In order to run locally, a dependancy to the ACR must be added to Helm. This can be done using the below command (replace `$(gpitfuturesdevacr-pass)` with a password from azure portal)
+In order to run locally, a dependancy to the ACR must be added to Helm. This can be done using the below command (replace `$(gpitfuturesdevacr-pass)` with the password from azure portal you obtained in k8s-private-registry.md)
```

Comment:
I think it would be highly beneficial for the reader if `k8s-private-registry.md` in the above sentense would link to the document. What do you think?

Reply:
Added required link in